lib/mayaUsd/utils/utilFileSystem.cpp

```diff
@@ -180,8 +180,8 @@ std::string UsdMayaUtilFileSystem::getPathRelativeToProject(const std::string& f
         return {};
 
     // Note: don't use isEmpty() because it is not available in Maya 2022 and earlier.
```

Comment:
Would have been nice if you removed this comment as its not relevant anymore since we are using std::string instead of MString.